assets/css/main.css

```diff
@@ -1,15 +1,37 @@
+@font-face {
+    font-family: gilroy-medium;
+    src: url("{{ site.baseurl }}/assets/font/Gilroy-Medium.otf") format("opentype");
+}
+
+
 html {
     height: 100%;
+    font-family: gilroy-semibold;
```

Comment:
Do we need to add a `@font-face` for **gilroy-semibold** or is it part of the one added above, **gilroy-medium**?

Reply:
this should be gilroy-medium, I just updated it, good catch as well